pkg/v1/sdk/features/Dockerfile

```diff
@@ -14,6 +14,7 @@ WORKDIR /workspace
 COPY go.mod go.mod
 COPY go.sum go.sum
 COPY apis/ apis/
+COPY cli/runtime cli/runtime
 RUN  go mod download
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
```

Comment:
Do we need to copy `cli/runtime` in the list of `COPY`s below?

Reply:
I saw failures in some package builds where this `runtime` library was getting used and hence added this line here.